packages/select/select.scss

```diff
@@ -119,7 +115,7 @@ $chevron-weight: rem(1px);
 
     &--no-value {
         .jkl-select__value {
-            color: $varm-fjellgra;
+            color: currentColor;
```

Comment:
Dette må nok avklares med design.

Reply:
Så ikke kommentaren din over før jeg kommenterte! Om dette skal inn bør det i så fall også oppdateres i Figma 😊

Reply:
Det bør nok også avklares om man skal gjøre tilsvarende i tekstfeltene! Høres ut som fin mat for designdagen på fredag 👩‍🎨🤓